docs/core/compatibility/configuration/7.0/diagnostics-config-section.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Breaking change: System.diagnostics entry in app.config"
+description: Learn about the .NET 7 breaking change in configuration where the `<section name="system.diagnostics">` entry is no longer allowed in an app.config file.
+ms.date: 11/03/2022
+---
+# System.diagnostics entry in app.config
+
+For applications that have an *app.config* file, the `<configuration><configSection>` entry may contain a `<section name="system.diagnostics">` entry, which is no longer allowed. If present, you must remove the entry.
```

Comment:
On L18, `<configSections>` (plural) is used. Are both uses valid? Or which one is correct?

Reply:
Good catch! I believe plural is correct.